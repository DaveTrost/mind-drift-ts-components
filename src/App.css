@import './styles/vars.css';

.container {
  max-width: 700px;
  margin: auto;
}
