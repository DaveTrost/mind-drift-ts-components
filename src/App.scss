@import './styles/vars.scss';

.container {
  max-width: 700px;
  margin: auto;
}
