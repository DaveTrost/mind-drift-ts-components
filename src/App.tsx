import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import Home from './components/Home';
import './App.css';

function App() {
  return (
    <div className={"container"}>
      <Router>
        <Switch>
          <Route exact path='/' component={Home} />
        </Switch>
      </Router>
    </div>
  );
}

export default App;
