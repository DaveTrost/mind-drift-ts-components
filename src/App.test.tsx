import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

// import withSessionMock from '../utils/WithSession';
// jest.mock('../utils/WithSession', () => ({}));

describe('App component', () => {
  it('renders App in logged-in state', () => {
    // withSessionMock.withSession = Comp => (<Comp />);
    
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  // it('renders App when not logged in', () => {
  //   withSessionMock.withSession = Comp => (<></>);

  //   const wrapper = shallow(<App />);
  //   expect(wrapper).toMatchSnapshot();
  // });
});
