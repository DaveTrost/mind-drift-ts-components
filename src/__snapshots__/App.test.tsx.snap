// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App in logged-in state 1`] = `
<div
  className="container"
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
    </Switch>
  </BrowserRouter>
</div>
`;
